### PR TITLE
feat: add be-web chart

### PR DESCRIPTION
**Pull request by gi0baro**
SSIA

### CODE REVIEW

**Review by edubacco — CHANGES_REQUESTED** [review]
also missing `PodMonitor` resource

File: charts/be-web/templates/worker.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      {{- end }}
+      {{- with .affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with $.Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+{{- end }}
```
[the remark this answers is not in the document]
Comment (gi0baro) [reply charts/be-web/templates/worker.yaml:102]: workers are not supposed to have endpoints..

File: charts/be-web/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{{- if .Values.ingress.enabled -}}
```
[the remark this answers is not in the document]
Comment (gi0baro) [reply charts/be-web/templates/ingress.yaml:1]: can we schedule this for later improvements?

File: charts/be-web/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+type: application
+name: be-web
+description: Generic web BE application
+
+version: 0.1.0
+appVersion: latest
```
Comment (edubacco) [inline charts/be-web/Chart.yaml:7]: As we cannot change this "on the fly" (with values.yaml), what about deleting this and use a custom field in `values.yaml`?

File: charts/be-web/templates/_helpers.tpl [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "app.baseLabels" -}}
+helm.sh/chart: {{ include "app.chart" . }}
+{{ include "app.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```
Comment (edubacco) [inline charts/be-web/templates/_helpers.tpl:72]: it's impossible to change the version without changing the chart (see comment in `Chart.yaml`)

File: charts/be-web/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+kubeVersion: ">=1.14.0-0"
```
[the remark this answers is not in the document]
Comment (gi0baro) [reply charts/be-web/Chart.yaml:8]: I think is still fine..

File: charts/be-web/templates/service.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.service.type }}
+  {{- with .Values.service.clusterIP }}
+  clusterIP: {{ . }}
+  {{- end }}
+  ports:
+    - name: http
```
[the remark this answers is not in the document]
Comment (gi0baro) [reply charts/be-web/templates/service.yaml:17]: I think is not so common, I'm not sure we can support this in a generic chart. I would try to make a dedicated chart for that..

**Pull request merged**